global/meta/models.yml

```diff
@@ -2806,6 +2807,26 @@ poll:
     read_only: true
     required: true
     restriction_mode: A
+  cryptographic_key:
+    type: string
```

Comment:
I suppose this and also following names are not visible for the customer, then I would prefer shorter names: `crypt_key`

Reply:
The keys are not visible. My first proposal was also "crypt_key". I changed this to have it consistent with the value of `poll/type`. @jsangmeister asked to change the value of `poll/type`. See the outdated discussion above.

Reply:
I have no strong opinion on the name of these fields, so we can leave it as it currently is